_pages/instructors-and-staff.html

```diff
@@ -19,6 +19,11 @@ <h2>Staff Biographies</h2>
         {% endif %}
         <p><strong>{{ employee.name }}</strong> {{ employee.bio }}</p>
     </div>
+    {% if employee.bio-2 %}
```

Comment:
What's the `-2` doing here?

Reply:
Not an issue just curious

Reply:
Some people have two bios: one in English and another in a different language. One individual's language is read right to left and cannot be included in the same `<p>` tag as their English bio.

Reply:
Gotcha, I see. Perhaps call this `bio_rtl` or something, and insert the RTL HTML `p` tag in the HTML rather than by including it under the YML string?

Reply:
I was considering this, but decided to opt for a more general `bio-2` tag because there is another person with a second bio, but doesn't need rtl text support. I'll change it to `bio_rtl`, since I haven't reworked the CSS to vertically center images alongside two paragraphs yet.